helpers.js

```diff
@@ -10,3 +10,28 @@ export const supportedTopicCategories = {
   1: { longName: "General", name: "general" },
   3: { longName: "Contributors", name: "contributors" },
 };
+
+export const displayRecentActivity = (bumpedAt) => {
+  const bumpDate = new Date(bumpedAt);
+  const timeDifference = Date.now() - bumpDate.getTime();
```

Comment:
```suggestion
export const formatDateDiff = (recent, old) => {
  const timeDifference = new Date(recent) - new Date(old);
```

Reply:
Makes sense. Definitely best to keep things as reusable as possible, and if/when we expand the project, having a site-wide date format will be helpful.